Sources/_FoundationInternals/String/String+Internals.swift

```diff
@@ -0,0 +1,19 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+extension String {
+    internal func _trimmingWhitespaces() -> String {
```

Comment:
I'd call this `_trimmingWhitespace()` which suggests that it's "all whitespace". Not a strongly held opinion.